css/floating-cart.css

```
@@ -0,0 +1,758 @@
+    max-width: none !important;
+    border-radius: var(--radius-lg) var(--radius-lg) 0 0;
+    margin: 0 !important;
+    padding: 0;
+    box-sizing: border-box;
+    box-shadow: 0 -10px 40px rgba(0, 0, 0, 0.15);
+    /* Ensure cart stays above mobile browser UI */
+    z-index: 9999 !important;
+    /* Add safe area padding for devices with notches/home indicators */
+    padding-bottom: env(safe-area-inset-bottom, 0);
+    /* Prevent horizontal scrolling */
+    max-width: 100vw;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

max-width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)